process_metrics_linux.go

```diff
@@ -89,8 +91,23 @@ func writeIOMetrics(w io.Writer) {
 	ioFilepath := "/proc/self/io"
 	data, err := ioutil.ReadFile(ioFilepath)
 	if err != nil {
-		log.Printf("ERROR: metrics: cannot open %q: %s", ioFilepath, err)
+		// Do not spam the logs with errors
+		// This error will not be fixed without process restart
+		var errMsg string
+		switch {
+		case os.IsNotExist(err):
```

Comment:
Have you considered trying `data, err := ioutil.ReadFile(statFilepath)`  via `init` function and log the error message only there. But omit it in this function which is called regularly? This supposes to simplify the logic...

Reply:
Thanks for the idea! Implemented this